mkdocs.yml

```diff
@@ -34,6 +34,7 @@ nav:
   - 'Intro': index.md
   - 'Tutorials': tutorials/index.md
   - 'Development': development/index.md
+  - 'Requirements': requirements/requirements.md
   - 'Explanation': explanation/index.md
```

Comment:
Missing link to hardware proposal

Reply:
Corrected in https://github.com/Dasharo/twpm-docs/pull/4/commits/f16f27b742b922fe820a54dea94d2eb168149a22

Reply:
I think both of the added files should be under `Explanation`. See https://github.com/Dasharo/twpm-docs/pull/7/commits/4c972960cdc5238288a7d84efeabe6902bb7bceb, it shows what should be done to create second level in menu.